virttest/env_process.py

```diff
@@ -1087,13 +1090,9 @@ def preprocess(test, params, env):
                     pvr,
                     remote_pvr,
                 )
-    # First, let's verify if this test does require root or not. If it
-    # does and the test suite is running as a regular user, we shall just
-    # throw a TestSkipError exception, which will skip the test.
```

Comment:
Just one comment, I suggest adding the above comments to the `class CheckRunningAsRoot(Setuper):` to keep the history.

Reply:
I removed it as I considered the called function name `verify_running_as_root` clear enough. Anyway, let me push that right now.

Reply:
Yeah, you're right, I stand on your side.